### ISSUE TITLE
Blue image selected box does not reset to top image when going to next video

### ISSUE THREAD

**Issue by jecava** [asalmon91/cull-AOSLO-images#9]
![cullimageserror](https://user-images.githubusercontent.com/70386435/92760728-8c038780-f356-11ea-91d9-5d1fef90eed5.PNG)
In attached example, I moved to the next video while the second image of the options in the previous video was selected. Now the blue box is still on the second option while the first option is bolded and shown. When I then try to view the second image it skips over it and tries to match up with the blue box.

**Comment by asalmon91** [comment]
Yeah these UI tables are difficult to work with. I'll work on a better option.

**Comment by jecava** [comment]
Thank you! More info: I just realized that blue box does not show up unless I manually click on an image to see it (which I do to avoid skipping images). So fixing those issues could be related.
